packages/katello/katello-installer-base/katello-installer-base.spec

```diff
@@ -118,6 +118,9 @@ ln -sf %{_datadir}/foreman-installer-katello/bin/katello-certs-check %{buildroot
 %doc README.*
 
 %changelog
+* Thu Oct 18 2018 Eric D. Helms <ericdhelms@gmail.com> - 3.9.0-0.2.rc1
```

Comment:
This now looks like a jump back because `3.9.0-2` > `3.9.0-0.2.rc1`. We can probably accept that for now but good to keep an eye on.